shell/common/engine_unittests.cc

```diff
@@ -162,7 +162,8 @@ TEST_F(EngineTest, Create) {
         /*animator=*/std::move(animator_),
         /*io_manager=*/io_manager_,
         /*font_collection=*/std::make_shared<FontCollection>(),
-        /*runtime_controller=*/std::move(runtime_controller_));
+        /*runtime_controller=*/std::move(runtime_controller_),
+        /*gpu_disabled_switch=*/nullptr);
```

Comment:
nit: make syncswitches here since technically it isn't supposed to be null.  We could also add a dcheck that it isn't null.